platform/android/export/export.cpp

```diff
@@ -2016,6 +2018,13 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			ret = unzGoToNextFile(pkg);
 		}
 
+		if (!invalid_abis.empty()) {
+			String unsupported_arch = String(", ").join(invalid_abis);
+			EditorNode::add_io_error("Missing libraries in the export template for the selected architectures: " + unsupported_arch + ".\n"
+				"Please build a template with all required libraries, or uncheck the missing architectures in the export preset.");
```

Comment:
My bad, my previous suggestion doesn't pass the `clang-format` style guidelines. This works:
```suggestion
			EditorNode::add_io_error(
					"Missing libraries in the export template for the selected architectures: " + unsupported_arch + ".\n" +
					"Please build a template with all required libraries, or uncheck the missing architectures in the export preset.");
```

Reply:
I think it worked with the previous suggestion, but it needed a lot of white spaces before the line "Please build a template..."
I tried deleting those white spaces and push without clang formatting. Then it failed.

Anyway, thanks a lot for the review and code suggestions !